src/main/java/machines/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator has started the job");
```

Comment:
The message 'Excavator has started the job' should be more informative and indicate that it is an Excavator that is working. Consider rephrasing to something like 'Excavator is working' to clearly reflect the type of Machine.